.github/workflows/requirements.yml

```diff
@@ -0,0 +1,119 @@
+name: Requirements
+
+on:
+  pull_request:
+    paths:
+      - requirements/core-requirements.yaml
+      - requirements/skinny-requirements.yaml
```

Comment:
Just to confirm, are we proposing to run this on pull requests that change requirements yaml files so that we can install the latest versions of any new libraries and determine whether or not there are compatibility issues?

Reply:
Changing the workflow name sounds good. How about `Validate Requirements`?